registries/_format/decimal.md

```diff
@@ -0,0 +1,31 @@
+---
+owner: baywet
+issue: 889
+description: A fixed point decimal number
+base_type: string
+layout: default
+remarks: Potential loss of precision when used with type number. Not specific enough about the size of the integral and fraction parts without the use of extensions.
```

Comment:
This is not specific to this format, it's simply a characteristic of JSON, and we don't want to get into the business of re-explaining JSON.

I don't think the registry should mention extensions that are not in the registry.

I'm going to take a stronger position and earlier and say that the registry should not be concerned with explaining numeric representation problems in JSON (which are discussed extensively in JSON-related forums, tutorials, etc.).

The role of the registry is to alert people to formats that are in use, to facilitate interoperability.  All this registration needs to do is note that the format has been used in a variety of conflicting ways and therefore is not interoperable. (although see my comment on line 4).

Reply:
removed the loss of accuracy, updated the comment to the interop aspect. @ralfhandl it might be worth registering your extensions. I understand the guidance is now to register namespaces. Do we expect namespace owners to then keep a registry of their extensions somewhere?